### DISCUSSION TITLE
How do I train on a translation dataset? (eg. {"english": "...", "chinese": "...")

### DISCUSSION THREAD

**Discussion by Nero10578** [axolotl-ai-cloud/axolotl#1046] [Q&A] [answered]
I am trying to fine tune mistral 7B on Sundanese and I am confused at how exactly did the Chinese-Alpaca fine tuners trained the model on their translation training dataset that can be found here: https://github.com/ymcui/Chinese-LLaMA-Alpaca/wiki/Training-Details#Training-Data

They showed an example dataset structure like so:
`{"english": "In Italy, there is no real public pressure for a new, fairer tax system.", "chinese": "在意大利，公众不会真的向政府施压，要求实行新的、更公平的税收制度。"}`

I ofcourse don't see the option of a dataset format that fits that, so do I just use completion raw corpus dataset type and put that inside the text key? Like so:

`{"text": ""english": "In Italy, there is no real public pressure for a new, fairer tax system.", "chinese": "在意大利，公众不会真的向政府施压，要求实行新的、更公平的税收制度。""}`

I am ofcourse going to use my own Sundanese dataset that I've made, but I am asking to figure out what's the best way to do this, so the model learns the translations the best. Any help would be appreciated! Thank you.

**Comment by NanoCode012** [comment, marked as answer]
In alpaca dataset, there are examples where the instruction is "Translate the following input from __ to ___"..

**Reply by Nero10578** [reply]
Thanks! Will try that but so far already getting success just giving it labeled raw text of each language. As well as just typical instruction dataset telling it to translate like in dolphin.

**Reply by NanoCode012** [reply]
Could you elaborate what you mean by labeled raw text?
